snapcraft/cli/remote.py

```diff
@@ -22,6 +22,7 @@
 from typing import List
 from xdg import BaseDirectory
 from . import echo
+from .config import enable_snapcraft_config_file
```

Comment:
why isn't this from ._config ?

Reply:
It is not meant to be used outside of the cli package, right?

Reply:
Correct, changed to _config.